src/main/kotlin/com/openlattice/organizations/Organization.kt

```diff
@@ -0,0 +1,145 @@
+package com.openlattice.organizations
+
+import com.fasterxml.jackson.annotation.JsonCreator
+import com.fasterxml.jackson.annotation.JsonIgnore
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.openlattice.apps.AppTypeSetting
+import com.openlattice.authorization.AclKey
+import com.openlattice.authorization.Principal
+import com.openlattice.authorization.PrincipalType
+import com.openlattice.client.serialization.SerializationConstants
+import com.openlattice.notifications.sms.SmsEntitySetInformation
+import com.openlattice.organization.OrganizationPrincipal
+import com.openlattice.organization.roles.Role
+import java.util.*
+
+/**
+ * Organization data object
+ * These are mutable for convenience, but the only way to persistently mutate is through entry processor.
+ * If you mutate these or pass around values do so at your own risk.
+ *
+ * @author Matthew Tamayo-Rios &lt;matthew@openlattice.com&gt;
+ */
+
+data class Organization(
+        val securablePrincipal: OrganizationPrincipal,
+        @JsonProperty(SerializationConstants.EMAILS_FIELD) val autoApprovedEmails: MutableSet<String>,
+        @JsonProperty(SerializationConstants.MEMBERS_FIELD) val members: MutableSet<Principal>,
+        @JsonProperty(SerializationConstants.ROLES) val roles: MutableSet<Role>,
+        @JsonProperty(SerializationConstants.SMS_ENTITY_SET_INFO)
+        val smsEntitySetInfo: MutableSet<SmsEntitySetInformation>,
+        @JsonProperty(SerializationConstants.PARTITIONS) val partitions: MutableList<Int> = mutableListOf(),
+        @JsonProperty(SerializationConstants.APPS) val apps: MutableSet<UUID> = mutableSetOf(),
+        val appConfigs: Map<UUID, Map<UUID, AppTypeSetting>> = mutableMapOf(),
```

Comment:
What is this field for? /does it need a @JsonProperty annotation?

Reply:
It was there before and wasn't used. I think I kept it there for you to use for your new app stuff, but I can remove if that is completely separate from organizations.

Reply:
I think think it's there on current develop 🤔

Reply:
At least it's not in Organization.java